notebooks/introduction_to_tensorflow/solutions/2_dataset_api.ipynb

```diff
@@ -4,25 +4,45 @@
    "cell_type": "markdown",
```

Comment:
<p>Nit: "Optimize data load and preprocessing with tf.data" should be title case since it is the title. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/706/discussion/#comment-2430738010'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>